ase/src/main/java/com/ase/ase/activemq/MessagingService.java

```diff
@@ -0,0 +1,81 @@
+package com.ase.ase.activemq;
+
+import org.apache.activemq.ActiveMQConnectionFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.Serializable;
+import java.util.Date;
+import javax.jms.Connection;
+import javax.jms.DeliveryMode;
+import javax.jms.JMSException;
+import javax.jms.MessageProducer;
+import javax.jms.Session;
+import javax.jms.TextMessage;
+import javax.jms.Topic;
+
+@Service
+public final class MessagingService {
+
+  @Value("${activemq.host}")
+  private String Host;
+
+  private MessageProducer           producer;
+  private ActiveMQConnectionFactory connectionFactory;
+  private Connection                connection;
+  private Session                   session;
+  private Topic                     destination;
+
+  public void send(Serializable content) {
+    try {
+      if (producer == null) {
+        init("new-data");
+      }
+      TextMessage message = session.createTextMessage(content.toString());
+      producer.send(message);
+    } catch (Exception e) {
+      System.out.println("Caught: " + e);
+      e.printStackTrace();
+    }
+  }
+
+  public void init(String queueName) throws JMSException {
+    connectionFactory = new ActiveMQConnectionFactory("tcp://" + Host + ":61616");
+    connectionFactory.setTrustAllPackages(true);
+    connection = createConnection(connectionFactory);
+    session = connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
```

Comment:
if connection is not possible there will be a Nullpointer Exeption. Maybe check this first